test/stubs/StubRuleSelection.ts

```diff
@@ -0,0 +1,20 @@
+import { OutputFormat, Rule, RuleSelection } from '@salesforce/code-analyzer-core';
+
+export class StubEmptyRuleSelection implements RuleSelection {
+    getCount(): number {
+        return 0;
+    }
+    getEngineNames(): string[] {
+        return ['EmptyEngine'];
+    }
+    getRulesFor(_engineName: string): Rule[] {
+        return [];
+    }
+    getRule(_engineName: string, _ruleName: string): Rule {
```

Comment:
I wasn't sure if this was worth implementing given that it doesn't have any rules 🤷‍♀️